incubator/hydra/templates/NOTES.txt

```diff
@@ -0,0 +1,16 @@
+{{- if eq .Values.config.system.secret "this_is_a_secret_change_me" }}
+######################################################################
+#                             WARNING                                #
+# You are using the default secret provided with the helm chart when #
+# issuing authorization tokens.  This is very insecure.              #
+######################################################################
+{{- end }}
+
+Hydra should not be facing the internet itself, so you can access hydra via. 
```

Comment:
No worries. Let me know if I can help :) I too need to set up Hydra. Figured I'd go big or go home.